src/init.cpp

```diff
@@ -1664,6 +1664,13 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (gArgs.IsArgSet("-seednode")) {
         connOptions.vSeedNodes = gArgs.GetArgs("-seednode");
     }
+    if (gArgs.IsArgSet("-connect")) {
+        // Initiate outbound connections unless connect=0
+        const auto connect = gArgs.GetArgs("-connect");
+        connOptions.bNoConnect = connect.size() == 1 && connect[0] == "0";
+        if (!connOptions.bNoConnect)
```

Comment:
Please rename this so that they're more clear, since this is exposed outside of CConnman now.
connOptions.vConnect -> connOptions.m_specified_outgoing
connOptions.bNoConnect -> (!) connOptions.m_use_addrman_outgoing